packages/frontend/amp/components/lib/GetImage.ts

```diff
@@ -0,0 +1,36 @@
+export const getImage = (
+    images: ImageSource[],
+    containerWidth: number,
+): SrcSet => {
+    const allImagesAllWeightings: SrcSet[] = images.reduce(
+        (acc: any[], curr: ImageSource) => {
+            acc.push(...curr.srcSet);
+            return acc;
+        },
+        [],
+    );
+
+    if (allImagesAllWeightings.length === 0) {
+        throw new Error('No inline images found');
+    }
```

Comment:
@AWare is this where you wanted the error to be? I feel this *should* just return `null` because some stories just don't have inline images

Reply:
Eg: this story would currently throw an Error on AMP because it doesn't have an in-Body image: https://www.theguardian.com/business/2018/dec/18/solar-power-energy-firms-government

Reply:
I think you're right - it doesn't seem appropriate to blow up here. But we're probably overdue a fuller talk on error-handling in the team so my suggestion is to stick with this for now until we have a better policy.

Reply:
fair enough!